src/math.ts

```diff
@@ -1,14 +1,14 @@
 import { BN } from "@coral-xyz/anchor";
 
+export const POINTS_PER_SECOND = new BN(10000);
 const MAX_U128 = new BN("340282366920938463463374607431768211455");
 const SECONDS_PER_LIQUIDITY_DECIMAL = 24;
 const LIQUIDITY_DECIMAL = 6;
 const LIQUIDITY_DENOMINATOR = new BN(10).pow(new BN(LIQUIDITY_DECIMAL));
 const SECONDS_PER_LIQUIDITY_DENOMINATOR = new BN(10).pow(
   new BN(SECONDS_PER_LIQUIDITY_DECIMAL)
 );
-
-export const POINTS_PER_SECOND = new BN(10000);
+const CUSTOM_DECIMAL = POINTS_PER_SECOND;
```

Comment:
It's better strict refer decimal to parameter. We are using it in context with secondsInside.
```ts
const SECONDS_INSIDE_DENOMINATOR = POINTS_PER_SECOND
```